Kernel/FileSystem/Ext2FileSystem.cpp

```diff
@@ -1208,9 +1208,61 @@ ErrorOr<void> Ext2FSInode::remove_child(StringView name)
     return {};
 }
 
-u64 Ext2FS::inodes_per_block() const
```

Comment:
Seems unintentional that this removes `inodes_per_block()` which will break the ALL_DEBUG build :)

Reply:
Whoops :yakkie: